code/lib/cli/src/initiate.ts

```diff
@@ -427,12 +427,16 @@ export async function initiate(options: CommandOptions): Promise<void> {
     logger.log('\nRunning Storybook');
 
     try {
-      const isReactWebProject =
-        projectType === ProjectType.REACT_SCRIPTS ||
-        projectType === ProjectType.REACT ||
-        projectType === ProjectType.WEBPACK_REACT ||
-        projectType === ProjectType.REACT_PROJECT ||
-        projectType === ProjectType.NEXTJS;
+      const supportsOnboarding = [
+        ProjectType.REACT_SCRIPTS,
+        ProjectType.REACT,
+        ProjectType.WEBPACK_REACT,
+        ProjectType.REACT_PROJECT,
+        ProjectType.NEXTJS,
+        ProjectType.VUE3,
+        ProjectType.NUXT,
```

Comment:
```suggestion
        ProjectType.NUXT,
```
Nuxt isn't working right now.